src/gitlab.js

```diff
@@ -78,6 +82,88 @@ class GitLab {
     });
   }
 
+  loadBranchs(cur_proj_name) {
+    var projId = encodeURIComponent(cur_proj_name);
+    m.request({
+      url: `${this.api_path}/projects/${projId}`,
+      method: "GET",
+      headers: {
+        "PRIVATE-TOKEN": this.private_token
+      }
+    }).then((proj) => {
+      this.cur_proj_name = proj.name_with_namespace;
+      this.cur_proj_id = proj.id;
+      this.loadTriggers();
+      this.branchs = null;
+      return this.loadBranchsBase(1, []);
+    }).catch((e) => {
+      alert(e);
+      return Promise.reject();
+    });
+  }
+  loadTriggers() {
+    m.request({
+      url: `${this.api_path}/projects/${this.cur_proj_id}/triggers`,
```

Comment:
It seems like that you perform `GET /projects/${projId}` for getting `project_id`, and you perform `GET /projects/${this.cur_proj_id}/triggers`. But this is redundant.

You can perform directly `GET /projects/${encodeURIComponent(cur_proj_name)}/triggers`

e.g) 

```bash
$ curl -s  --header "PRIVATE-TOKEN: $GITLAB_API_PRIVATE_TOKEN" "https://gitlab.com/api/v4/projects/sue445%2Fgitlab_awesome_release/triggers" | jq .
[]
```

see. https://docs.gitlab.com/ce/api/pipeline_triggers.html#list-project-triggers

Reply:
So, all the ":id" can use ${encodeURIComponent(cur_proj_name)} ? 

I find that documents of trigger use the numeric id, so i assume only numeric id works.

https://docs.gitlab.com/ce/ci/triggers/README.html